shlr/meson.build

```diff
@@ -1,12 +1,99 @@
-res = run_command(py3_exe, '-c', '__import__("sys").exit(__import__("os").path.exists("capstone"))')
-if res.returncode() == 0
-  if not git_exe.found()
-    error('Cannot load capstone library. Either provide capstone in ./shlr/capstone or install git, so it can be downloaded')
+capstone_dep = dependency('capstone', version: '>=3.0.4', required: false)
```

Comment:
Hmm. This one looks too old to me:
```

Version 3.0.4

@aquynh aquynh released this on Jul 15, 2015 · 421 commits to master since this release
```
Better to make it at least 3.0.5: https://github.com/aquynh/capstone/releases
And also ping him one more time on a 4.0 release.

Reply:
ok